src/functTest/groovy/com/bmuschko/gradle/docker/tasks/image/DockerRemoveImageFunctionalTest.groovy

```diff
@@ -42,6 +42,75 @@ class DockerRemoveImageFunctionalTest extends AbstractGroovyDslFunctionalTest {
         !result.output.contains("repository")
     }
 
+    def "can remove multiple images"() {
```

Comment:
We'll also want to change all documentation and test cases to use the new method instead.